modules/installation-vsphere-infrastructure.adoc

```diff
@@ -6,21 +6,25 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6.5
-or 6.7U2 or later instance.
+You must install the {product-title} cluster on a VMware vSphere version that meets the requirements for the components that you use.
 
-VMware recommends using vSphere Version 6.7 U2 or later with your {product-title}
-cluster.
-vSphere 6.7U2 includes:
+.Minimum supported vSphere version for VMware components
+|===
+|Component | Minimum supported versions |Description
 
-* Support for VMware NSX-T
-* Support for vSAN, VMFS and NFS, using the in-tree VCP
+|Hypervisor
+|vSphere 6.5 with HW version 13
+|This version is the is the minimum version that {op-system-first} supports. See the link:https://access.redhat.com/ecosystem/search/#/ecosystem/Red%20Hat%20Enterprise%20Linux?sort=sortTitle%20asc&vendors=VMware&category=Server[Red Hat Enterprise Linux 8 supported hypervisors list].
```

Comment:
@dav1x 
Do u know where can I check the Hypervisor version of vcsa1? And does qe has this privilege?